CHANGELOG.asciidoc

```diff
@@ -17,6 +17,7 @@ https://github.com/elastic/apm-server/compare/4daa36bd5c144cf9182afc62dc8042af66
 ==== Added
 
 - Enriched data with IP and UserAgent {pull}393[393].
+- Remove usage of beats normalizer for beat.Events {pull}687[687]
```

Comment:
this is an implementation detail, i don't think is something users need to be aware of.